### PR TITLE
updating inert

### PR DESCRIPTION
**Pull request by rachelandrew**
Updating `HTMLElement.inert` as Chrome is shipping in 102. Also added `inert` as a global attribute. 

https://chromestatus.com/feature/5703266176335872

### CODE REVIEW

**Comment by nt1m** [comment]
Not sure with safari_ios is set to false, but this is also supported on iOS (same version as desktop).

**Comment by nt1m** [comment]
Also, Firefox recently enabled it by default only on Nightly/early betas: https://bugzilla.mozilla.org/show_bug.cgi?id=1764134 in version 101, but keeps it disabled for release versions, not sure how to reflect that.

**Comment by rachelandrew** [comment]
@nt1m I've update iOS, as far as Firefox goes I think the extra level of detail isn't that useful as in the stable version it's behind a flag, which is probably what people think of looking at  this.

File: html/global_attributes.json [hunk 1]
```diff
@@ -1143,6 +1143,83 @@
...
+            "firefox_android": {
+              "version_added": "81",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "html5.inert.enabled",
+                  "value_to_set": "true"
+                }
+              ]
+            },
```
Comment (queengooborg) [inline html/global_attributes.json:1179]: Firefox Android actually doesn't have flag support anymore!

```suggestion
            "firefox_android": {
              "version_added": false
            },
```

**Review by queengooborg — CHANGES_REQUESTED** [review]
Thanks for your PR!  Just a small detail, but this is LGTM otherwise!

**Comment by queengooborg** [comment]
@nt1m If the feature is available in nightly but disabled in beta/stable, we use ["preview" for the version](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#choosing-preview-values).  I'd be happy to review a separate PR to make those changes, or if you'd like to make the changes here @rachelandrew, I'm also happy to merge them!

**Comment by rachelandrew** [comment]
@queengooborg I've committed your change and also made the preview update.

**Review by queengooborg — APPROVED** [review]
Thank you, this is LGTM!

**Pull request merged**